apps/fft/fft.hpp

```diff
@@ -266,93 +274,60 @@ class signal
     std::cout << "]" << std::endl;
   }
 
-  bool isFFT(signal &X, int threads = std::thread::hardware_concurrency())
+  bool isFFT(signal &X, scheduler auto sch, int maxN = 20000)
   {
     int N = y.size();
     bool ret = true;
 
-    data_t *Y = new data_t[N];
-    data_t * M  = new data_t[N*N];
-    auto A = std::mdspan<data_t, view_2d, std::layout_right>(M, N, N);
+    //int nparts = N/maxN;
+    //int psize = std::min(N, nparts);
+    //int matsize = psize * psize;
```

Comment:
we have commented and used statements here.

Reply:
removed these